backend/routes/topTenToppings.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from 'express'
+import { getToppingsCollection } from '../db'
+import expandToppingsCategories from '../expandToppingsCategories'
+import { Topping, ToppingWithCategoryId } from '../topping'
+import { SuccessResponse } from '../response'
+
+export default async function topTenToppings(
+  _: Request, response: Response<SuccessResponse<Topping[]>>)
+{
+  const toppingsCollection = await getToppingsCollection()
+  const rawToppings =
+    await toppingsCollection
+      .find<ToppingWithCategoryId>({})
+      .sort( { "winRatio": -1 } )
```

Comment:
how is "winRatio: -1" supposed to work again? wouldn't the win ratio be between 0 & 1?